packages/office-addin-dev-certs/src/install.ts

```diff
@@ -10,7 +11,7 @@ import { isCaCertificateInstalled, verifyCertificates } from "./verify";
 function getInstallCommand(caCertificatePath: string, machine: boolean = false): string {
    switch (process.platform) {
       case "win32":
-         return `powershell -ExecutionPolicy Bypass scripts\\install.ps1 ${machine ? "LocalMachine" : "CurrentUser"} '${caCertificatePath}'`;
+         return `powershell -ExecutionPolicy Bypass ${path.resolve(__dirname, "..\\scripts\\install.ps1")} ${machine ? "LocalMachine" : "CurrentUser"} '${caCertificatePath}'`;
```

Comment:
Suggestion, use separate variable to compose the script path to simplify reading the command line:

```
const script = `${path.resolve(__dirname, "..\\scripts\\install.ps1")}`;
return `powershell .....   ${script} ......`;
```